ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -49,7 +49,14 @@ let default_vbd_backend_kind = ref "vbd"
 
 let ca_140252_workaround = ref false
 
-let xen_platform_pci_bar_uc = ref true
+(* Optimize performance: set MTRR WB attribute on Xen PCI MMIO BAR.
+   This is useful for AMD, and mostly a noop on Intel (which achieves a similar
+   effect using Intel-only features in Xen)
```

Comment:
The part in brackets is going to go stale quickly.  In order to support encrypted memory (and later, encrypted VMs), we're going to have to stop using iPAT on Intel, at which point Intel will become like AMD and this control will be relevant.

Reply:
ok, so doing this uniformly now on both Intel and AMD actually future proofs us against that. You can then change the Intel side from iPAT in Xen without further toolstack changes?